cohorts.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: page
+title: OLS Cohorts
+image: /images/syllabus.jpg
+photos:
+  name: Niklas Morberg
+  license: CC BY-NC 2.0
+  url: https://flic.kr/p/5BXB6s
+---
+
+# Cohorts
+
+<div class='cohort-grid'>
+  {% for cohort in site.cohorts %}
+    <div class="flip-card">
+      <div class="flip-card-inner">
+        <div class="flip-card-front">
+          <h1>{{ cohort.name }}</h1>
+        </div>
+        <div class="flip-card-back">
+          <a href="/{{ cohort.link }}"><h2>Click here to find out more about the {{ cohort.number }} cohort!</h2></a>
+        </div>
+      </div>
+    </div>
+ {% endfor %}
+</div>
+
+# OLS Cohorts - An Overview
+
+<!-- OLS-1 -->
```

Comment:
This part could move at the top of the page and then you could use the information extracted there in the card (like dates, etc)

Reply:
I dont understand
I should move the overview up and the cards down but i should still include the things in the overview on the cards? should i just pick some or all?

Reply:
Or do you just want me to move "OLS COHORTS - AN OVERVIEW" to the top of the page and not the whole table

Reply:
![Screenshot (47)](https://user-images.githubusercontent.com/102422875/197284933-f33c5126-6734-495c-9df1-2cfaa2755cda.png)
With this picture, am i on track with what you want me to do? Any way it would fit better with what you have in mind?
If i do include all the information from the table into the cards, will the table still be needed?